Swiftfin/Views/SettingsView/UserDashboardView/UsersView/Components/UsersRow.swift

```diff
@@ -0,0 +1,162 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import Factory
+import JellyfinAPI
+import SwiftUI
+
+extension UsersView {
+
+    struct UsersRow: View {
+
+        @Injected(\.currentUserSession)
+        private var userSession
+
+        @Default(.accentColor)
+        private var accentColor
+
+        // MARK: - Environment Variables
+
+        @Environment(\.colorScheme)
+        private var colorScheme
+
+        @CurrentDate
+        private var currentDate: Date
+
+        private var user: UserDto
+
+        // MARK: - Actions
+
+        private let onSelect: () -> Void
+        private let onDelete: () -> Void
+
+        // MARK: - User Status Mapping
+
+        private var userActive: Bool {
+            if let isDisabled = user.policy?.isDisabled {
+                return !isDisabled
+            } else {
+                return false
+            }
+        }
+
+        // MARK: - Initializer
+
+        init(
+            user: UserDto,
+            onSelect: @escaping () -> Void,
+            onDelete: @escaping () -> Void
+        ) {
+            self.user = user
+            self.onSelect = onSelect
+            self.onDelete = onDelete
+        }
+
+        // MARK: - Label Styling
+
+        private var labelForegroundStyle: some ShapeStyle {
+            userActive ? .primary : .secondary
+        }
+
+        // MARK: - User Image View
+
+        @ViewBuilder
+        private var userImage: some View {
+            ZStack {
+                ImageView(user.profileImageSource(client: userSession!.client))
+                    .pipeline(.Swiftfin.branding)
+                    .placeholder { _ in
+                        SystemImageContentView(systemName: "person.fill", ratio: 0.5)
+                    }
+                    .failure {
+                        SystemImageContentView(systemName: "person.fill", ratio: 0.5)
+                    }
+                    .grayscale(userActive ? 0.0 : 1.0)
+            }
+            .squarePosterStyle()
+            .posterShadow()
+            .frame(width: 60, height: 60)
+        }
+
+        // MARK: - Row Content
+
+        @ViewBuilder
+        private var rowContent: some View {
+            HStack {
+                VStack(alignment: .leading) {
+
+                    Text(user.name ?? L10n.unknown)
+                        .font(.headline)
+                        .lineLimit(2)
+                        .multilineTextAlignment(.leading)
+
+                    TextPairView(
```

Comment:
I don't think this is that essential of information that it should be on every row. I do think it would be good to show something if the user is disabled though, but I have a different design in mind with a badge.

Reply:
I've removed this TextPair. Do you want me to take a shot at adding an icon for this or do you want me to leave this as is?

Reply:
I just tried my design and didn't entirely like it, I think I'm fine with just not showing that row.

Reply:
It looks like web just makes the disabled users grayscale so we're mirroring that functionality. An icon would be a nice to have but I think just the grayscale should be enough. I have that all on my latest commit.